docs/examples/hairpin_fitting/hairpin_unfolding.rst

```diff
@@ -0,0 +1,289 @@
+    )
+
+Fit the result of the fit::
+
+    plt.figure()
+    baseline_1x.plot(label="baseline f1x")
+    baseline_2x.plot(label="baseline f2x")
+    plt.ylabel('Force (pN)')
+    plt.legend()
+
+.. image:: baselinefit.png
+
+Compute the piezo distance
+--------------------------------
+
+Now that we have determined the distance calibration and fitted the baseline, the piezo distance can be computed::
+
+    piezo_calibration = lk.PiezoForceDistance(distance_calibration, baseline_1x, baseline_2x, signs=(-1,1)) 
+
+Chose an fd curve to compute the Piezo distance for::
+
+    rv = lk.File(filenames[3])
+
+    tether_length_rv, corrected_force_1x_rv, corrected_force_2x_rv = piezo_calibration.force_distance(
+    rv['Trap position']['1X'], rv.force1x, rv.force2x, downsampling_factor=500
+    )
+    force_data_rv = corrected_force_2x_rv
+
+Plot the result. To check that the computation of the Piezo Distance was succesful, we compare it to the camera tracking distance::
+
+    plt.figure()
+    plt.scatter(tether_length_rv.data, force_data_rv.data, s=1, label = 'Piezo Distance')
+    rv.fdcurves['FRETHP_bp9_Fd1_try2_rv'].plot_scatter(label = 'fd curve')
+    plt.legend()
+    plt.xlabel('Distance [$\mu$m]')
+    plt.ylabel('Force [pN]')
+
+.. image:: piezodistance.png
+
+Fit the data
+-----------------
+
+Next, we extract the contour length of the unfolded hairpin by fitting the data before and after the unfolding event.
+
+
+Data Selection
+^^^^^^^^^^^^^^
+
+First, select data before and after the unfolding event::
+
+    folded_min = 1.45 #lower lim (in um) for FD curve with folded hairpin
+    folded_max = 1.67 #upper lim (in um) for FD curve with folded hairpin
+    unfolded_min = 1.68 #lower lim (in um) for FD curve with unfolded hairpin
+    unfolded_max = 1.8 #upper lim (in um) for FD curve with unfolded hairpin
+
+    force_data_rv_data = force_data_rv.data
+    tether_length_rv_data = tether_length_rv.data
+
+    force_back_folded = force_data_rv_data[(tether_length_rv_data < folded_max) & (tether_length_rv_data > folded_min)]
+    distance_back_folded = tether_length_rv_data[(tether_length_rv_data < folded_max) & (tether_length_rv_data > folded_min)]
+
+    force_back_unfolded = force_data_rv_data[(tether_length_rv_data < unfolded_max) & (tether_length_rv_data > unfolded_min)]
+    distance_back_unfolded = tether_length_rv_data[(tether_length_rv_data < unfolded_max) & (tether_length_rv_data > unfolded_min)]
+
+Plot the selected data::
+
+    plt.figure()
+    plt.scatter(distance_back_folded, force_back_folded,s=2,alpha=0.2,label='Selection folded')
+    plt.scatter(distance_back_unfolded, force_back_unfolded,s=2,alpha=0.2,label='Selection unfolded')
+    plt.legend()
+    plt.ylabel('Force (pN)')
+    plt.xlabel('Distance ($\mu$m)')
+
+.. image:: selected_data.png
+
+Define the models
+^^^^^^^^^^^^^^^^^
+
+Define the model for the DNA with folded hairpin::
+
+    dna_handles_force = lk.ewlc_odijk_force("dna_handles")
```

Comment:
The force regime we are in here is quite low, which might stress the validity of the Odijk model a bit (which is intended for 10 < pN < 30). I wonder if that is part of the reason why we are seeing the deviations in `Lp`.

Reply:
but the part after unfolding does go up to 30 pN.

Reply:
Yeah. I didn't mean you had to change anything. I'm just wondering out loud why the persistence length is different from what I'd expect.